src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_errors.c

```diff
@@ -60,6 +57,11 @@ xocl_fini_errors(struct xocl_dev_core *core)
 	struct xcl_errors *err = core->errors;
 	if (!err)
 		return;
-	//TODO
+
+	mutex_init(&core->errors_lock);
+	xocl_clear_all_error_record(core);
```

Comment:
please remove this line because the memory is freed by the following vfree.

Reply:
done